src/main/kotlin/pt/up/fe/ni/website/backend/service/ActivityService.kt

```diff
@@ -32,4 +36,59 @@ abstract class ActivityService<T : Activity>(
         activity.teamMembers.removeIf { it.id == idAccount }
         return repository.save(activity)
     }
+
+    fun addHallOfFameMemberById(idActivity: Long, idAccount: Long): T {
+        val activity = getActivityById(idActivity)
+        val account = accountService.getAccountById(idAccount)
+        if (activity.teamMembers.find { it.id == idAccount } != null) {
+            // TODO: deal with member that already exists on the team, should it return an error or move?
```

Comment:
Should it throw an error or move the account when adding to the hall of fame while existing on the team.